pkg/cmd/cli/cmd/deploy.go

```diff
@@ -350,7 +350,7 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig) error {
 				continue
 			}
 
-			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = deployapi.DeploymentCancelledAnnotationValue
+			deployment.Annotations[deployapi.DeploymentCancelledAnnotation] = unversioned.Now().Format(time.RFC3339)
```

Comment:
We're using `time.Now` as well, iow. server time is the one presented to the user.